src/main/java/hudson/plugins/analysis/core/HealthAwareRecorder.java

```diff
@@ -237,6 +247,10 @@ public HealthAwareRecorder(final String healthy, final String unHealthy,
 
     // CHECKSTYLE:ON
 
+    public HealthAwareRecorder(String pluginName) {
+        this.pluginName = "[" + pluginName + "] ";
+    }
+
```

Comment:
This was done as it is recommended here: https://github.com/jenkinsci/workflow-plugin/blob/master/COMPATIBILITY.md#constructor-vs-setters